packages/mesa/dri-meson.build.patch

```diff
@@ -0,0 +1,11 @@
+--- src/src/gallium/targets/dri/meson.build	2023-02-09 00:05:36.000000000 +0300
++++ src/src/gallium/targets/dri/meson.build.patch	2023-02-16 11:50:28.886282708 +0300
+@@ -132,7 +132,7 @@
+         'devenv_@0@'.format(d),
+         input : libgallium_dri,
+         output : d,
+-        command : [prog_ln, '-f', '@INPUT@', '@OUTPUT@'],
++        command : [prog_ln, '-fs', '@INPUT@', '@OUTPUT@'],
+         build_by_default : true,
+       )
+     endforeach
```

Comment:
This patch seems to break installation. We handle hardlinks in `termux_step_post_make_install` of `build.sh` and so this patch is not needed. (Strictly this results in on-device building not being supported though.)